pkg/helm/option.go

```diff
@@ -339,6 +341,20 @@ func InstallReuseName(reuse bool) InstallOption {
 	}
 }
 
+// InstallSubNotes will (if true) instruct Tiller to render SubChart Notes
+func InstallSubNotes(enable bool) InstallOption {
+	return func(opts *options) {
+		opts.subNotes = enable
```

Comment:
by making a call to `InstallSubNotes`, it is also instructing tiller to render subcharts' notes during a `helm upgrade`. It'd be better to move the `subNotes` variable into the installReq and upgradeReq structs so they can be isolated similar to `upgradeDescription`.

Reply:
From Helm's perspective, behaviorally speaking nothing will change. From the SDK's point of view, it blocks this  from happening:

```
res, err := client.InstallReleaseFromChart(
    ...,
    helm.InstallSubNotes(true)
)
...
res, err := client.UpgradeReleaseFromChart(
    ...
)
```

which will cause the subcharts to render when `client.UpgradeReleaseFromChart` is called.

Does that make sense?

Reply:
This should be resolve with latest push.